### DISCUSSION TITLE
Duplicate button for flows

### DISCUSSION THREAD

**Discussion by prasanna31** [postmanlabs/postman-flows#287] [Ideas]
Duplication of flows will help us if we want the same flow with some minor changes. 
This will reduce the burden of building the flow from scratch.

**Comment by dev-kanishk** [comment]
Sure @prasanna31 , We’ll add this right away, and with the next release, flows will have a duplicate option.

**Reply by prasanna31** [reply]
When can we expect this release @dev-kanishk

**Reply by samay-v** [reply]
Hey @prasanna31, this is available now, could you try updating your app?

**Reply by prasanna31** [reply]
Thank you @samay-v . I can see it now.
